crates/driver/src/domain/liquidity/mod.rs

```diff
@@ -1,8 +1,6 @@
-pub use self::fetcher::Fetcher;
 use crate::domain::eth;
 
 pub mod balancer;
-pub mod fetcher;
```

Comment:
I moved this to `infra::liquidity` mostly because:
- Liquidity fetching is "infrastructure"-y (in the same way as the `blockchain::Ethereum` module is)
- There is a solidifying plan to push liquidity indexing into a separate service, at which point it will truly fit in here.